docs/modules/circulation/pages/billing.adoc

```diff
@@ -39,7 +39,14 @@ A credit can be kept in a patron account and used later to pay bills.
 
 To use patron credit to pay bills, select *Patron Credit* as the payment type.
 
-If your library does not use patron credit, you can turn it off through a library setting.
+If your library does not use patron credit, you can turn it off through a library setting: "Disable Patron
+Credit".
+
+[TIP]
+===========================================================================
+Check out xref:negative_balances.adoc[] for information on finding
```

Comment:
```suggestion
Check out xref:local_admin:negative_balances.adoc[] for information on finding
```
Looks like you need to [specify the module](https://docs.antora.org/antora/latest/navigation/xrefs-and-link-text/#page-xrefs) if you are linking to a page in a different module.